### DISCUSSION TITLE
Contributor call: 6 August 2022

### DISCUSSION THREAD

**Discussion by l3xander** [freesewing/freesewing#2416] [Support] [answered]
You are all invited to the FreeSewing Contributor Call.
Please leave a comment here or [on Discord](https://discord.freesewing.org/) to add things to the agenda.
This issue will be updated after the call with notes and to-dos.

### About
FreeSewing's contributor call is an informal hangout/chat/meeting that is open to all FreeSewing contributors and users alike.
The call is organized/hosted by @tangerineshark.
The call does not have a fixed structure or agenda items. Everyone is free to suggest agenda items, ask questions, or bring ideas.

Details:
Location: https://discord.freesewing.org (meeting room: 🔊voice chat)
Date: 6 August 2022 4pm UTC, which can be found in your local time zone [here](https://savvytime.com/converter/utc-to-belgium-antwerpen-ca-los-angeles-il-chicago-ny-new-york-city-brazil-sao-paulo-united-kingdom-london-france-paris-south-africa-johannesburg-msd-india-mumbai-hkt-japan-tokyo-australia-sydney/aug-6-2022/4pm).
Or check the bot in Discord! #meetings
The meetings will always be at 6pm, Antwerp time.

### Agenda
- We could use a note-taker if anyone's game!
- Improving how parts are added to designs - speak now! https://github.com/freesewing/freesewing/discussions/2538 (maybe @joostdecock and/or @eriese can lead this)
- Who do you want to hear from in the next newsletter? And what else would you like to read?
- TBD

> This discussion will be updated after the call with notes and to-dos.

**Comment by eriese** [comment]
Maybe this would be a good time to discuss #2538? Like at a high level, what are the goals for a future way of dealing with parts, what are the frustrations with the current way of dealing with parts

**Reply by tangerineshark** [reply]
Love this idea - adding!

**Comment by KaerMorhan** [comment, marked as answer]
**Notes**

Participants:

- @joostdecock 
- @tangerineshark
- @lexander2002 
- @KaerMorhan
- @bbdapalkow
- @bobgeorgethe3rd 
- @eriese 
- @eleonoremilbou
- @woutervdub 
- Benjamin F.
- Austin (oyhoiby)
- Starf
- Rick

**JOOST&ERIESE**
Improving how parts are added to the designs.

> Short review of the discussion [#2538](https://github.com/freesewing/freesewing/discussions/2538): patterns are made from different parts and it’s difficult to add new parts. FreeSewing strives to let the users make the layout themselves. The issue with adding new parts comes into play when certain pieces are cut on fold, for example, and the user might want to unfold the piece and cut it flat (so a new part would be needed). A lot of users are not experienced coders so the idea is to implement a system which would help users to customize their layout easier and more efficiently.

One of the proposals is to have one file to configure the pattern parts and their layout. The system should probably be more designer-oriented rather than coder-oriented since it should be possible for a designer with zero coding experience to design and code a pattern for the website.
Eriese’s proposal is to put all the configuration into the pattern file. Later parts can use previously available information and pull from previous configuration. The compromise would be to implement per part configuration. In that way parts will automatically have all the previously available information tied to it. Magic!

_Bobgeorgethe3rd’s feedback: it’s hard to code in multiple pieces since it takes several different files, and it is something that Joost addresses in his discussion. Disagreeing with the idea of not needing to look into the core configuration of the parts since it would grant a deeper understanding and control of the pattern. It might prevent some damage done to the first-piece configuration._

_Wouter’s feedback: as a sewer primarily with some experience in code, it is might be useful to have an idea and then ask coders about the technicalities of implementing it._

_Joost’s feedback: it is better knowing what is going on under the hood but making it a requirement would only gatekeep new designers from contributing to the website._

**ELEONORE**
Inclusive design for people with disabilities. Feedback from Eleonore’s friend who is a wheelchair user.
Different things to think about for permanent wheelchair users and part-time wheelchair users. Implementing patterns that have configurative waistband height would greatly improve the experience of wheelchair users. Pictures of garments on a sitting model would also help. For people with scoliosis it would be useful to configure only one side of the garment.

_Feedback: The last thing can be possible by splitting the code into two different pieces or a pattern that supports multiple drafts. Having two measurement profiles for users would work but also it might be worthwhile to make shoulder slope measurements different in the first place since it is a common thing present in many bodies. There are also people with missing limbs so some parts might need to be hidden. It is possible to do that with the new developing system of configuring the layout of the pattern (unnecessary parts will not show to the user)._

_Morgan's feedback: people with osteoporosis and similar conditions might need to add more room in the back and less in the front._

**REMINDERS for the future:**

- Conditional loading of patterns
- To mark the pattern as right, left or shared

**Comment by joostdecock** [comment]
Closing to increase visibility of active discussions
